src/licensedcode/data/licenses/vsftpd-openssl-exception.LICENSE

```diff
@@ -0,0 +1,19 @@
+---
+key: vsftpd-openssl-exception
+short_name: vsftpd OpenSSL exception
+name: vsftpd OpenSSL exception
+category: Copyleft Limited
+owner: Richard Cochran
+is_exception: yes
+spdx_license_key: vsftpd-openssl-exception
+other_urls:
+    - https://git.stg.centos.org/source-git/vsftpd/blob/f727873674d9c9cd7afcae6677aa782eb54c8362/f/LICENSE
+    - https://launchpad.net/debian/squeeze/+source/vsftpd/+copyright
+    - https://github.com/richardcochran/vsftpd/blob/master/COPYING
+---
+
+vsftpd is licensed under version 2 of the GNU GPL.
```

Comment:
This replaces [`gpl-2.0_with_generic-exception_7.RULE`](https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/gpl-2.0_with_generic-exception_7.RULE)
but the text has `is licensed under version 2 of the GNU GPL.` should this not be removed, as this is not part of the exception text? (Which is why the rule license expression was `gpl-2.0 WITH generic-exception` and not just `generic-exception`)

Reply:
ok as it is, and owner value is correct.